vdom/core.py

```diff
@@ -8,19 +8,33 @@
 This module provides the core implementation for the VDOM (Virtual DOM).
 
 """
-def toJSON(el):
-    """Convert an element to JSON"""
-    if(type(el) is str):
-        return el
+
+from jsonschema import validate, Draft4Validator, ValidationError
+
+def toJSON(el, schema=None):
```

Comment:
I love that you made the `schema` optional here so I can sidestep if my layouts are already validated. 😄